apps/vscode/command_client/command_client.py

```diff
@@ -88,11 +82,9 @@ def to_dict(self):
 def write_request(request: Request, path: Path):
     """Converts the given request to json and writes it to the file, failing if
     the file already exists unless it is stale in which case it replaces it
-
     Args:
         request (Request): The request to serialize
         path (Path): The path to write to
-
```

Comment:
again

Reply:
Done

Reply:
doesn't look fixed? still deleting lines here

Reply:
Double checked should be fixed.